src/promptflow/tests/test_configs/flows/web_classification_with_additional_include/prepare_examples.py

```diff
@@ -1,6 +1,9 @@
 from pathlib import Path
 from promptflow import tool
 
+# read file from additional includes
+lines = open(r"fetch_text_content_from_url.py", "r").readlines()
```

Comment:
Will this line work if it's inside tool function code?

Reply:
It will work if it is in the function code.
It is recommended to Xiaole to move this type of code to function code, but because he is not the owner of the code, he cannot change it. So we fixed this issue on the SDK side